python/lsst/ip/isr/isrTask.py

```diff
@@ -574,6 +582,10 @@ def run(self, ccdExposure, bias=None, linearizer=None, dark=None, flat=None, def
                 self.maskAndInterpNan(ccdExposure)
                 interpolationDone = True
 
+            if self.config.brighterFatterLevel == 'CCD':
+                bfKernel = bfKernel[ccdExposure.getDetector().getId()]
+            else:
+                raise NotImplementedError("per-amplifier brighter-fatter correction not yet implemented")
```

Comment:
Is this outdated? Your new code in `cp_pipe` does support per-amplifier brighter-fatter correction.

Reply:
It's actually not outdated, no. The code there generates kernels at the per-amplifier level, however this is the correction code, and nothing has yet been written to support doing the _correction_ at the per-amplifier level. If it looked trivial I would have chucked it in with this ticket, but because the detector is already assembled at this point in the code it's not quite that easy. I will file a ticket saying that this needs to be added at some point though.